main.go

```diff
@@ -220,7 +223,11 @@ func main() {
 		w := tabwriter.NewWriter(os.Stdout, 1, 1, 1, ' ', 0)
 		fmt.Fprintln(w, "Masked Email\tFor Domain\tState\tLast Email At\t")
 		for _, email := range maskedEmails {
-			fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", email.Email, email.ForDomain, email.State, email.LastMessageAt)
+			if email.State == "deleted" && ! *flagShowDeleted {
+				continue
+			} else {
```

Comment:
nits - this else branch isn't necessary, the continue above is already short-circuiting here

Reply:
I've changed it in 2 subsequent commits.